### PR TITLE
feat: [TKC-2623] testkube kind Docker image

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

Docker image for running Testkube Agent in Docker (OSS and Cloud)

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Comment by olensmar** [comment]
wow! should this be in its own repo? would it be possible to pass env/org/agent ids to connect to a control plane?

**Comment by vsukhin** [comment]
@olensmar yes, just decided to store it somewhere. yes, sure, it's just a few docker args

File: build/kind/cloud/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+fi
+
+# Check if env id is provided
+if [ -z "$ENV_ID" ]; then
+  log "Please provide ENV_ID env var"
+  exit 1
+fi
+
+# Check if cloud url is provided
+if [ -z "$CLOUD_URL" ]; then
```
Comment (rangoo94) [inline build/kind/cloud/entrypoint.sh:30]: Isn't that rather `ROOT_DOMAIN`?

File: build/kind/oss/Dockerfile.kind [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+RUN curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+# Step 6: Install Testkube CLI
+RUN curl -sSLf https://get.testkube.io | bash
+
+# Step 7: Script to automatically create Kind cluster and install Testkube
+COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+RUN chmod +x /usr/local/bin/entrypoint.sh
+
+# Step 8: Example K6 Test Workkflow CRD and preload Kind images
```
Comment (rangoo94) [inline build/kind/oss/Dockerfile.kind:28]:
```suggestion
# Step 8: Example K6 Test Workflow CRD and preload Kind images
```

File: build/kind/cloud/entrypoint.sh (hunk 1 above)
Comment (vsukhin) [reply build/kind/cloud/entrypoint.sh:30]: I just followed exsitng agent helm installation command  `--set testkube-api.cloud.url=agent.testkube.io:443`, will ROOT_DOMAI will be clear for user?

**Pull request merged**